app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+  def update_likes_count
+    posts.likes_counter += 1
+  end
```

Comment:
## Optional
- [ ] I highly suggest that you use [Rails active callback](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the #update_likes_count method every time a new like has been saved to the database, then make the update_likes_count method private